packages/react-components/react-utilities/src/hooks/useTimeout.ts

```diff
@@ -1,4 +1,12 @@
 import { useBrowserTimer } from './useBrowserTimer';
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+
+const setTimeoutNoop = (callback: Function) => {
```

Comment:
can this be simplified to be truly a noop ?

```ts
const setTimeoutNoop = () => -1;
const clearTimeoutNoop = () => undefined;
```

any kind of extra logic provided here shouldn't be needed